ci-operator/step-registry/cucushift/winc/prepare/cucushift-winc-prepare-commands.sh

```
@@ -254,13 +297,32 @@ echo "Windows OS image ID: $windows_os_image_id"
 # Get replica count
 winworker_machineset_replicas=$(oc get machineset -n openshift-machine-api $winworker_machineset_name -o jsonpath="{.spec.replicas}")
 
-echo "Waiting for Windows nodes to come up in Running state"
+echo "DEBUG: Windows machineset configuration:"
+oc get machineset -n openshift-machine-api ${winworker_machineset_name} -o yaml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)